src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -134,7 +136,10 @@ export default class ProcessConverter extends AbstractConverter<Process> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private getEventDefinitions(bpmnElement: any): EventDefinition[] {
     return bpmnEventKinds.map(eventKind => {
-      return { kind: eventKind, counter: ensureIsArray(bpmnElement[eventKind + 'EventDefinition']).length };
+      // sometimes eventDefinition is simple and therefore it is parsed as empty string "", in that case eventDefinition will be converted to an empty object
```

Comment:
:warning: we should add a test if this is a real issue, otherwise this should be removed

Reply:
yes tests are in process - needed to discuss with you the changes and agreed on the solution